.github/workflows/simple-tests.yml

```diff
@@ -0,0 +1,37 @@
+# workflow for running quick, simple tests in the simulator. Runs on every push
+name: Simple Tests
+
+on:
+  push:
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install tools
+      run: |
+        sudo apt -y update
+        sudo apt -y install gcc-arm-none-eabi
+    - name: Initialize submodules
+      run: git submodule update --init --recursive
+    - name: clean
+      run: make clean
+    - name: make simulator
+      run: make simulator
+    # simple test for verifying testing infrastructure
+    # run tests in sudo for rtprio to work
+    - name: Unit Test 
+      run: |
+        sudo ./Validation/test.sh critical_overcurrent
+        sudo ./Validation/test.sh critical_overtemp_discharging
+        sudo ./Validation/test.sh critical_overvolt
```

Comment:
We will never be in the critical state with an overcurrent because the contactors will be off (if we are then we are kinda screwed). I will remove it from the yml file.

Reply:
We will never be in the critical state and charging since the contactors will be closed. It is the same for critical_overcurrent, but the code wouldn't work with charging because we initialize the BPS by setting the initial current reading to 0.